src/Components/Components/src/PublicAPI.Unshipped.txt

```diff
@@ -13,11 +13,11 @@ Microsoft.AspNetCore.Components.Routing.NotFoundEventArgs.Path.get -> string!
 Microsoft.AspNetCore.Components.Infrastructure.ComponentStatePersistenceManager.ComponentStatePersistenceManager(Microsoft.Extensions.Logging.ILogger<Microsoft.AspNetCore.Components.Infrastructure.ComponentStatePersistenceManager!>! logger, System.IServiceProvider! serviceProvider) -> void
 Microsoft.AspNetCore.Components.Infrastructure.ComponentStatePersistenceManager.SetPlatformRenderMode(Microsoft.AspNetCore.Components.IComponentRenderMode! renderMode) -> void
 Microsoft.AspNetCore.Components.Infrastructure.RegisterPersistentComponentStateServiceCollectionExtensions
-Microsoft.AspNetCore.Components.SupplyParameterFromPersistentComponentStateAttribute
-Microsoft.AspNetCore.Components.SupplyParameterFromPersistentComponentStateAttribute.SupplyParameterFromPersistentComponentStateAttribute() -> void
-Microsoft.Extensions.DependencyInjection.SupplyParameterFromPersistentComponentStateProviderServiceCollectionExtensions
+Microsoft.AspNetCore.Components.PersistentStateAttribute
+Microsoft.AspNetCore.Components.PersistentStateAttribute.PersistentStateAttribute() -> void
+Microsoft.Extensions.DependencyInjection.PersistentStateProviderServiceCollectionExtensions
```

Comment:
This needs to move to the `Microsoft.AspNetCore.Components.Infrastructure` namespace